### ISSUE TITLE
"Submitting jobs" syntax is confusing

### ISSUE THREAD

**Issue by xylar** [ACME-Climate/a-prime#16]
There are several cases where the a-prime user sees output like:
```
Submitting jobs to compute seasonal climatology for 20171122.beta3rc10_1850.ne30_oECv3_ICG.edison
```
I found this wording confusing when I am running on the login node, since my impression is that no job is being submitted to the queue.  I would change "Submitting jobs to compute..." to simply "Computing ..." for clarity.

**Comment by xylar** [comment]
Should be addressed by #24

**Comment by milenaveneziani** [comment]
addressed by #24
